src/vs/workbench/parts/terminal/electron-browser/terminalActions.ts

```diff
@@ -262,8 +262,10 @@ export class CreateNewTerminalAction extends Action {
 		if (event instanceof MouseEvent && (event.altKey || event.ctrlKey)) {
 			const activeInstance = this.terminalService.getActiveInstance();
 			if (activeInstance) {
-				this.terminalService.splitInstance(activeInstance);
-				return Promise.resolve(null);
+				return activeInstance.getCwd(this.terminalService.configHelper).then(cwd => {
+					this.terminalService.splitInstance(activeInstance, { cwd: cwd });
```

Comment:
You can use the shorthand here of `{ cwd }` because the key and variable are the same